package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@pwnfinance/style-config",
+  "version": "1.1.0",
+  "description": "Shared ESLint, StyleLint and Prettier configuration for PWN TypeScript projects.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/PWNFinance/style-config.git"
+  },
+  "author": "PWN",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/PWNFinance/style-config/issues"
+  },
+  "homepage": "https://github.com/PWNFinance/style-config#readme",
+  "peerDependencies": {
+    "eslint": "^8.11.0"
+  },
+  "dependencies": {
+    "@semantic-release/changelog": "^6.0.2",
+    "@semantic-release/git": "^10.0.1",
+    "@typescript-eslint/eslint-plugin": "^5.16.0",
+    "@typescript-eslint/parser": "^5.16.0",
+    "@vue/eslint-config-typescript": "^11.0.0",
+    "babel-eslint": "^10.1.0",
+    "eslint": "^8.11.0",
```

Comment:
maybe move this to `devDependencies` + keep in `peerDependencies`? also similar with `prettier` and `stylelint`... however do not burn too much time on this if it won't work, just thoughts

Reply:
iirc I tried this when I was setting it up and it caused a lot of unnecessary problems so I will keep this the way it is if it's not causing any other problems I don't know about.